src/overlays/actors/ovl_En_Guruguru/z_en_guruguru.c

```diff
@@ -21,32 +45,358 @@ const ActorInit En_Guruguru_InitVars = {
     (ActorFunc)EnGuruguru_Update,
     (ActorFunc)EnGuruguru_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Guruguru_0x80BC6BF0/EnGuruguru_Init.asm")
+u16 textIDs[] = { 0x292A, 0x292B, 0x292C, 0x292D, 0x292E, 0x292F, 0x2930, 0x2931,
+                  0x2932, 0x2933, 0x2934, 0x2935, 0x2936, 0x294D, 0x294E };
+
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_NONE,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_2,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK0,
+        { 0x00000000, 0x00, 0x00 },
+        { 0xF7CFFFFF, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
+    { 15, 20, 0, { 0, 0, 0 } },
+};
+
+static AnimationHeader* D_80BC79CC[] = { &D_06000B04, &D_0600057C };
+static u8 D_80BC79D4[] = { 0 };
+static f32 D_80BC79D8[] = { 1.0f, 1.0f };
+static void* D_80BC79E0[] = { D_06005F20, D_06006320 };
+static void* D_80BC79E8[] = { D_06006720, D_06006920 };
```

Comment:
Since we know those are eyes and mouth textures, you can name the array accordingly.
```suggestion
static void* sEyeTextures[] = { D_06005F20, D_06006320 };
static void* sMouthTextures[] = { D_06006720, D_06006920 };
```